config/adminlte.php

```diff
@@ -128,12 +128,12 @@
         [
             'text' => 'profile',
             'url'  => 'admin/settings',
-            'icon' => 'user',
+            'icon' => 'fa fa-fw fa-user',
```

Comment:
This sould be `fas`(solid) or `far`(regular/mostly outline) `fa` is deprecated.

> ```
> <i class="fa fa-camera-retro"></i>  <!-- version 4's syntax -->
> 
> <i class="fas fa-camera-retro"></i> <!-- version 5's syntax -->
> ```
This is a part of the [FA5 Upgrade Guide](https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4)

Reply:
was fixed